.editorconfig

```diff
@@ -0,0 +1,13 @@
+# https://editorconfig.org
+
+root = true
```

Comment:
which editor does this belong to? where is this needed? maybe this should be in the gitignore?

Reply:
No. It doesn't ignore anything. It enforces other things that eslint overlooks. Read about it at https://editorconfig.org it works in loads of editors. We've also been gradually adding them to repos to help keep things tidy/consistent.